var/spack/repos/builtin/packages/powerapi/package.py

```diff
@@ -11,6 +11,7 @@ class Powerapi(AutotoolsPackage):
     homepage = "https://powerapi.sandia.gov/"
     git      = "https://github.com/pwrapi/pwrapi-ref.git"
 
+    version('20200130', commit='21f75b1469261d99e604f7ddc18f30513ebdd048')
```

Comment:
```suggestion
    version('2020-01-30', commit='21f75b1469261d99e604f7ddc18f30513ebdd048')
```
This lets you say something like `depends_on('powerapi@2020:')`.

Reply:
Done!